### DISCUSSION TITLE
Advent Calendarの全体ランキングからはOrganizationは別扱いにしていただきたい

### DISCUSSION THREAD

**Discussion by ishiayaya** [increments/qiita-discussions#392] [要望・アイデア・不具合]
Organizationは企業に興味がなかったら、全体あっても全く行こうとも思いません。
Organizationのイメージもかえって悪くなるのではないでしょうか。
Organizationのない全体ランキングがあるといいなと考えます。

**Comment by tanaka-asami415** [comment]
@ishiayaya 
ご意見ありがとうございます！
来年以降の改善の参考にさせていただきたいので、もう少し詳しくお聞かせいただけると幸いです🙇

> Organizationは企業に興味がなかったら、全体あっても全く行こうとも思いません。

Organizationに興味がない場合でも、個別の記事には興味を持つこともあると思うのですが
Organizationカレンダーという時点で、閲覧対象から外れてしまうということでしょうか？

> Organizationのイメージもかえって悪くなるのではないでしょうか。

興味がないOrganizationをランキングで見ることにより嫌いになってしまう
というような理解でいいでしょうか？🙇

**Comment by ishiayaya** [comment]
@tanaka-asami415 
> Organizationに興味がない場合でも、個別の記事には興味を持つこともあると思うのですが`

個別の記事にメインテーマがあるのであれば、そのテーマのカテゴリーのAdvent Calenderに各でしょう。
OrganizationのAdvent Calnederは「Organizationに所属している人しか書けない」のですから、
そのOrganizationに関することを書くのだと、通常考えるのではないでしょうか。

> Organizationカレンダーという時点で、閲覧対象から外れてしまうということでしょうか？

AIのある会社もの、HRのある会社のもの、と興味がなかったら、外れます。

そいいう話ではありません。私はMicrosoft の人間ではりませんが、「Microsoft Power BI」のカレンダーをコミュニティ活動として、たてました。のサービスの良さを広く知ってほしいという思いからです。

そしてランキングをQiitaが提供しているのであれば、「このテーマはホットなんだな」ということを広く知ってもらうために、あると考えています。そこに、組織のものが入ってくる、購読者が少ないのに、いいねはやたらと多い、という現状が入ると「組織票ですか？」「組織名を知らしめたいのですか？」「広告の代わりに頑張っているのですか」とどうしても見えてしまうのです。

個人で行っているものと、異質なものを同じ土俵のランキングにカウントの仕方が同じだからという理由で並べている、そして組織としていいねを頑張った人だけが上位にいる、というように見えます。

> | Organizationのイメージもかえって悪くなるのではないでしょうか。
> 興味がないOrganizationをランキングで見ることにより嫌いになってしまう
> というような理解でいいでしょうか？🙇

上記のようなうがった見方をしてしまうような人（私も含めて）もいるでしょう、ということですがわかりませんか、感覚が。周囲の知人とも意見交換しましたが、同じ意見でした。

**Comment by ishiayaya** [comment]
組織は組織通し、頑張りたかったら頑張ればよいでしょうし、その他のカテゴリーと組織は全く別物のいいねのつき方がするので、分けては？と申し上げています。
@tanaka-asami415

**Reply by tanaka-asami415** [reply]
@ishiayaya 
詳細を教えていただきありがとうございます！
来年以降の改善の参考にさせていただきます🙇

**Reply by P-SiZK** [reply]
@ishiayaya 
大変お待たせいたしました :bow:

ishiayaya さんにいただいた内容や、インタビュー等でいただいた他のユーザーさんからの意見も検討し、
より全員にとって納得感のあるランキングページを目指し、カレンダーランキングのページを「各カテゴリーごとのランキングを表示するページ」に変更しました！
Organizationのカレンダーも「Organization」というカテゴリーに分かれてランキングが表示されるようになるため、
今回提起いただいた課題も解決できたのではないかと考えております。

- https://qiita.com/advent-calendar/2024/ranking

![image](https://github.com/user-attachments/assets/ddda2c57-a8e4-4599-89ae-1cd71f4207c6)

**Reply by ishiayaya** [reply]
@P-SiZK 大変ありがとうございます!
